.github/workflows/nightly.yml

```diff
@@ -328,3 +328,25 @@ jobs:
           AWS_ACCESS_KEY_ID: "${{ secrets.CI_AWS_ACCESS_KEY_ID }}"
           AWS_SECRET_ACCESS_KEY: "${{ secrets.CI_AWS_SECRET_ACCESS_KEY }}"
         run: make release-helm
+
+        nightly-failure:
+          name: nightly-failure
+          if: failure()
+          needs:
+            - build-x86_64-unknown-linux-gnu-packages
+            - build-x86_64-unknown-linux-musl-packages
+            - build-aarch64-unknown-linux-musl-packages
+            - build-x86_64-apple-darwin-packages
+            - build-x86_64-pc-windows-msvc-packages
+            - release-docker
+            - release-s3
+            - release-cloudsmith
+            - release-helm
+          runs-on: ubuntu-latest
+          steps:
+          - name: Discord notification
+            env:
+              DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
+            uses: Ilshidur/action-discord@master
+            with:
+              args: 'simple-keyboard - Nightly Build Failed :('
```

Comment:
I think we could provide a link to the failed job with something like:

`args: "Nightly failed: https://github.com/${{github.repository}}/commit/${{github.sha}}/checks"`

https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#github-context

Either way, I assume `simple-keyboard` was copy/pasta? :smile: